pkg/domain/infra/abi/network.go

```diff
@@ -12,10 +13,36 @@ import (
 )
 
 func (ic *ContainerEngine) NetworkList(ctx context.Context, options entities.NetworkListOptions) ([]types.Network, error) {
+	var filterDangling bool
+	var wantDangling bool
+
+	// dangling filter is not provided by netutil
+	if val, ok := options.Filters["dangling"]; ok {
+		filterDangling = true
+		switch strings.ToLower(val[0]) {
```

Comment:
I recommend to use https://pkg.go.dev/strconv#ParseBool instead

Reply:
much better, thanks